documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -0,0 +1,246 @@
+// Category: debezium-using
+// Type: assembly
+// ModuleID: configuring-debezium-to-auto-create-change-data-capture-topics
+// Title: Configuring {prodname} to use automatically create topics
+[id="cdc-topic-auto-create-config"]
+= Custom Topic Auto-Creation
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+{prodname} automatically creates *internal* topics for offsets, connector status, config
+storage and history topics. The destination topics for the captured tables will be
+automatically created with a default config by the Kafka brokers when
+`auto.create.topics.enable` is set to `true`.{empty} +
+When topic creation is disabled on the brokers, for example in production environments,
+or when the topics need a different configuration then these topics have to be created
+upfront either automated in a custom deployment process or manually until Kafka Connect 2.6.
+
+Since Kafka 2.6.0 Kafka Connect supports customizable topic auto-creation.
+
+== Kafka Connect
+
+Kafka Connect since Kafka 2.6.0 comes with topic creation enabled:
+
+[source,options="nowrap",shell]
+----
+topic.creation.enable = true
+----
+
+ifdef::community[]
+[NOTE]
+====
+If you don't want to allow automatic topic creation by connectors you can set this value to `false`
+in the Kafka Connect config (_connect-distributed.properties_ file or via environment variable
+_CONNECT_TOPIC_CREATION_ENABLE_ when using https://hub.docker.com/r/debezium/connect[{prodname}'s container image for Kafka Connect]).
+====
+endif::community[]
+
+ifdef::product[]
+[NOTE]
+====
+If you don't want to allow automatic topic creation by connectors you can set this value to `false`
+in the Kafka Connect CRD:
+====
+
+[source,yaml,options="nowrap"]
+----
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+
+...
+
+spec:
+  config:
+    topic.creation.enable: "false"
+----
+endif::product[]
```

Comment:
is the above a correct example for it? WDYT? @gunnarmorling

Reply:
In general, yes, it is. Problem though is that downstream isn't on AK 2.6 yet, so it doesn't really make sense to have this here yet. But it's not a problem, as this page will not be added to downstream docs without explicit opt-in. Can you please log a Jira issue for tracking that for 1.3. Thanks!

Reply:
@TovaCohen  already created https://issues.redhat.com/browse/FUSEDOC-4157 to handle adding and styling of that docs section. It's also already annotated for the downstream module handling.

Reply:
Ah, just seeing below this issue already exists. All good then.